src/python/espressomd/reaction_ensemble.pyx

```diff
@@ -720,10 +769,14 @@ cdef class WidomInsertion(ReactionAlgorithm):
 
     def measure_excess_chemical_potential(self, reaction_id=0):
         """
-        Measures the excess chemical potential in a homogeneous system.
+        Measures the excess chemical potential in a homogeneous system for the provided reaction_id.
+        Please register the insertion moves first via calling the method add_reaction (with only product types specified).
         Returns the excess chemical potential and the standard error for the
-        excess chemical potential. It assumes that your samples are
-        uncorrelated in estimating the standard error.
+        excess chemical potential. The error estimate assumes that your samples are
+        uncorrelated.
```

Comment:
This assumption is almost never fulfilled, so the provided error estimates are useless in most cases. What is worse, the output gives no chance to the user to check how well the assumption was fulfilled. I think that it would be more useful to output the whole time-series, and let the user estimate the error using his own favorite method.

Reply:
This goes beyond documentation. Can you open a PR which fixes the corresponding issue #3283 ?

Reply:
I am afraid that I cannot open such PR within next 10 days because I am busy with other staff.

Reply:
This is part of another issue, resolving it here.